datastream-utils/src/main/java/com/linkedin/datastream/common/zk/ZkClient.java

```diff
@@ -47,20 +47,34 @@
    * @param zkServers the ZooKeeper connection String
    * @param sessionTimeout the session timeout in milliseconds
    * @param connectionTimeout the connection timeout in milliseconds
+   * @param operationRetryTimeoutMs The maximum amount of time, in milli seconds, each failed
+   *                                operation is retried. A value lesser than 0 is considered as
+   *                                retry forever until a connection has been reestablished.
    */
-  public ZkClient(String zkServers, int sessionTimeout, int connectionTimeout) {
-    super(zkServers, sessionTimeout, connectionTimeout, new ZKStringSerializer());
 
+  public ZkClient(String zkServers, int sessionTimeout, int connectionTimeout, int operationRetryTimeoutMs) {
```

Comment:
Done.

Reply:
I've left the DEFAULTS without MS since they are public to avoid updating all dependencies. Since the variable names have been updated with the ms, it offers some improvement in readability as opposed to none for the sake of consistency. And there is some consistency still with all variables having the ms, while all defaults left without it.